.mailmap

```diff
@@ -272,6 +272,7 @@ Alexander Dunlap <alexander.dunlap@gmail.com>
 Alexander Eberspächer <alex.eberspaecher@gmail.com>
 Alexander Hirzel <alex@hirzel.us>
 Alexander Pozdneev <pozdneev@users.noreply.github.com>
+Alexander Zhura <nice.zhura@list.ru>
```

Comment:
You need to run the `.mailmap` file by the following command `bin/mailmap_check.py` after adding your name and email.
https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file

Reply:
The problem was that I committed changes in web GitHub and it placed wrong author name. I previously added myself into this list and everything worked. Helpfully with git amend and rebase I fixed such problem and now everything should work. Please check error messages better next time.